test/unit/vm/json_schema/validation_error_test.dart

```diff
@@ -0,0 +1,608 @@
+// This test suite verifies that validation errors report correct values for
+// instance & schema paths.
+
+import 'package:json_schema/json_schema.dart';
+import 'package:test/test.dart';
+
+// TODO: test nested $refs
```

Comment:
Handle this here or create a new ticket?

Reply:
https://jira.atl.workiva.net/browse/WAP-2634